.github/workflows/30min-iiab-test-install-raspios.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           cpu_info: ${{ matrix.cpu_info }}
           copy_repository_path: /opt/iiab/iiab
           commands: |
-              test `uname -m` = ${{ matrix.arch }}
+              #test `uname -m` = ${{ matrix.arch }}
```

Comment:
@neomatrixcode please explain quick why these lines are being commented out?

Reply:
I copied the test command from the example, but I don't consider it important. I consider removing that line but I just put it in a comment

Reply:
Would output sometimes be `test aarch64 = aarch64` and sometimes (something like!) `test x86_64 = x86_64` ?

(Of so obvious to gurus — but can we try to make our work more approachable & maintainable for newcomers/others?)

Reply:
Would output sometimes be test aarch64 = aarch64 and sometimes (something like!) test x86_64 = x86_64 ?
yes, that's correct, uname -m returns the name of the platform the operating system is running on